redwood-treehouse/src/hostMain/kotlin/app/cash/redwood/treehouse/EventPublisher.kt

```diff
@@ -38,9 +38,8 @@ internal class EventPublisher(
     listener.appCanceled(app)
   }
 
-  val ziplineEventListener = object : ZiplineEventListener() {
+  fun ziplineEventListener(app: TreehouseApp<*>) = object : ZiplineEventListener() {
```

Comment:
FYI @JakeWharton @colinrtwhite this is new; I’m tracking this eagerly rather than trying to infer it from events.